lib/ClassMover/Adapter/TolerantParser/TolerantClassReplacer.php

```diff
@@ -103,13 +112,16 @@ private function classIsTheOriginalInstance(ClassReference $classRef, FullyQuali
         return $classRef->isClassDeclaration() && $classRef->fullName()->equals($originalName);
     }
 
-    private function addUseStatement(TextDocument $source, FullyQualifiedName $newName): TextEdits
+    private function addUseStatement(Code $source, FullyQualifiedName $newName): TextEdits
     {
-        return $this->updater->textEditsFor(SourceCodeBuilder::create()->use($newName->__toString())->build(), Code::fromString($source->__toString()));
+        return $this->updater->textEditsFor(SourceCodeBuilder::create()->use($newName->__toString())->build(), $source);
     }
 
-    private function addNamespace(TextDocument $source, QualifiedName $qualifiedName): TextEdits
+    private function addNamespace(Code $source, QualifiedName $qualifiedName): TextEdits
     {
-        return $this->updater->textEditsFor(SourceCodeBuilder::create()->namespace($qualifiedName->__toString())->build(), Code::fromString($source->__toString()));
```

Comment:
note `Code::` need probably be entirely replcaed by TextDocument, so this type of refatoring isnt really necessaary